tox.ini

```diff
@@ -0,0 +1,58 @@
+[pytest]
+filterwarnings =
+    error
+    ; Suppress warnings about an import of `imp` by Pyramid
+    ignore:the imp module is deprecated in favour of importlib
+
+[tox]
+envlist = py36-tests
+skipsdist = true
+minversion = 3.8.0
+requires =
+    tox-pip-extensions
+    tox-pyenv
+    tox-run-command
+tox_pip_extensions_ext_venv_update = true
+tox_pyenv_fallback = false
+
+[testenv]
+skip_install = true
+setenv =
+    dev: NGINX_SERVER={env:NGINX_URL:http://localhost:9081}
+    dev: H_EMBED_URL = {env:H_EMBED_URL:http://localhost:5000/embed.js}
+    dev: VIA_URL = {env:VIA_URL:http://localhost:9080}
+passenv =
+    HOME
+    dev: CHROME_EXTENSION_ID
+    dev: SENTRY_DSN
+deps =
+    tests: coverage
+    {tests,docstrings,checkdocstrings}: pytest
+    {tests,docstrings,checkdocstrings}: factory-boy
+    {tests,docstrings,checkdocstrings}: mock
+    lint: flake8
+    {format,checkformatting}: black
+    {format,checkformatting}: isort
```

Comment:
I think you need an isort config file like this: https://github.com/hypothesis/lms/blob/master/.isort.cfg